contrib/freebsd/files/patch-meson.build

```diff
@@ -0,0 +1,11 @@
+--- meson.build.orig	2021-05-31 11:30:38.106093380 +0200
++++ meson.build	2021-05-31 11:31:11.774182473 +0200
+@@ -252,7 +252,7 @@
+ libgcab = dependency('libgcab-1.0', version : '>= 1.0', fallback : ['gcab', 'gcab_dep'])
+ gcab = find_program('gcab', required : get_option('tests'))
+ bashcomp = dependency('bash-completion', required: false)
+-python3 = find_program('python3')
++python3 = find_program('python3.8')
```

Comment:
any cleaner way to do this?  Maybe something like this upstream?
```
if !python3.found()
    python3 = find_program('python3.8')
endif
```

Or instead have a for loop of options (python3, python3.8, python3.9) or something?

Reply:
Added.